docs/docs/alerting.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Alerting
+weight: 13
+---
+{{% pageinfo color="primary" %}}
+Alerting resources require Grafana version 9.5 or higher.
+{{% /pageinfo %}}
+
+The Grafana Operator currently only supports _Grafana Managed Alerts_.
+
+For data source managed alerts, refer to the documentation and tooling available for the respective data source.
+{{% alert title="Note" color="primary" %}}
+When using Mimir/Prometheus, you can use the [`mimir.rules.kubernetes`](https://grafana.com/docs/agent/latest/flow/reference/components/mimir.rules.kubernetes/) component of the Grafana Agent to deploy rules as Kubernetes resources.
```

Comment:
OMG, I didn't know about this feature. This just got bumped to the top of our TODO internally <3